Sources/ArcGISToolkit/Components/FloatingPanel/FloatingPanel.swift

```diff
@@ -84,7 +84,7 @@ struct FloatingPanel<Content>: View where Content: View {
                     }
                     content
                         .frame(height: height)
-                        .padding(.bottom, isCompact ? 25 : .zero)
+                        .padding(.bottom, isCompact ? 25 : 10)
```

Comment:
If `isCompact` is false, `10` provides an amount of content bottom padding that is similar to the amount of padding around the handle.

<img width="158" alt="Screenshot 2022-12-08 at 14 21 49" src="https://user-images.githubusercontent.com/16397058/206579691-49f7f689-647c-46f2-9c33-3c09f0eae6b8.png">

If `isCompact` is true, we continue to use `25`. This values helps any content avoid layout conflicts with the iOS Home Indicator. On devices without the Home Indicator, `25` is more than needed but SwiftUI doesn't provide a method to detect the top of the Home Indicator so we need to use a value that works for both conditions.

<img width="133" alt="Screenshot 2022-12-08 at 13 58 00" src="https://user-images.githubusercontent.com/16397058/206580906-2058f727-7876-4f8e-a3e0-de13705af831.png">
<img width="187" alt="Screenshot 2022-12-08 at 13 58 53" src="https://user-images.githubusercontent.com/16397058/206580908-84272d58-2a85-401e-ada6-8f8fd0590e7f.png">